services/web/src/helpers/screen.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { startCase } from 'lodash';
+import { Helmet } from 'react-helmet-async';
+import { APP_NAME } from 'utils/env';
+
+export default function(Component) {
```

Comment:
thats weird why nest it like this ?

Reply:
not sure what you mean by "nest"? It's a hoc...